api/api/models/moderation.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth.models import User
+from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+
+class UserPreferences(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
```

Comment:
Barring a few exceptions and old code, the API seems to conventionally use strings to reference target in relationships.

```suggestion
    user = models.OneToOneField(to="auth.User", on_delete=models.CASCADE)
```

Reply:
Updated in 6762fae2db5a62a65136f9d9d74a73f0e0a64d49